src/main/scala/org/ergoplatform/nodeView/history/storage/HistoryStorage.scala

```diff
@@ -58,8 +55,8 @@ class HistoryStorage private(indexStore: LDBKVStore, objectsStore: LDBKVStore, c
     objectsStore.get(idToBytes(id)).map(_.tail) // removing modifier type byte with .tail
```

Comment:
ScalaDoc missing here and for other methods

Reply:
done